test/Microsoft.Kusto.ServiceLayer.UnitTests/Microsoft.Kusto.ServiceLayer.UnitTests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
STS has been updated to netcore3.1 - you should merge up to main and then update this to match the rest

Reply:
And that being said - what I would suggest is you separately do the merge from main and get that merged into your feature branch first so this PR stays concise.